### PR TITLE
Fixed some minor docs issues in BigQuery

### PR DESCRIPTION
**Pull request by evildour**

### CODE REVIEW

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@
 namespace Google.Cloud.BigQuery.V2
 {
     /// <summary>
-    /// A dataset within BigQuery.
+    /// An asynchronous task within BigQuery.
```
Comment (jskeet) [inline apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs]: I'd rather use "job" here than task - asynchronous task sounds like it's a `Task<T>`.

Comment (evildour) [inline apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs]: Done

**Comment by evildour** [comment]
PTAL

**Comment by jskeet** [comment]
Thanks - feel free to merge on green.

**Comment by evildour** [comment]
Oh, never mind. I didn't see you comment about it being fine.

**Pull request merged**